README.md

```diff
@@ -1 +1,51 @@
-# dat
+# Delta Acceptance Testing (DAT)
+
+The DAT project makes it easy to generate Delta Lake reference tables and run integration tests to check different implementations of the Delta Lake PROTOCOL behave as expected.
+
+The Delta Lake implementations can clone this repo, generate the reference tables, and implement the suggested integration tests in the language of their choice.
+
+## Installation
+
+You can install the dependencies for this project by running `poetry install`.  Run `poetry shell` to activate the virtual environment.    If `poetry shell` does not activate the environment on your machine, you may need to run this command:
+
+```
+source "$( poetry env list --full-path | grep Activated | cut -d' ' -f1 )/bin/activate"
+```
+
+Run `python -m dat.main` to make sure everything is working properly.
+
+## How to build tables
+
+Start by removing the `out` directory with `rm -rf out`.
+
+Create the reference tables with `python -m dat.main write-generated-reference-tables`.
+
+This will create reference tables as follows:
+
+```
+out/
+  tables/
+    generated/
+      reference_table_1/
+        delta/
+          ...
+        parquet/
+          table_content.parquet
+        table_metadata.json
+      reference_table_2/
+        ...
+```
+
+## Example integration tests
+
+Here's an example integration test:
+
+```python
+import chispa
+
+actual_df = spark.read.format("delta").load("out/tables/generated/reference_table_1/delta")
+expected_df = spark.read.format("parquet").load("out/tables/generated/reference_table_1/parquet")
+chispa.assert_df_equality(actual_df, expected_df)
```

Comment:
why are we comparing the `delta` vs `parquet` reference tables?

isn't the point to compare the reference table (i.e. "expected") with the output of YOUR client (e.g. rust delta client) table (i.e. "actual")?

Reply:
We're checking to make sure that the Delta Lake Table contains the data we expect it to contain with the PySpark reader.  The `actual_df` contains the Delta Lake contents per the PySpark reader.  The `expected_df` contains the data we expect the Delta Lake table to contain.  `out/tables/generated/reference_table_1/parquet` was originally going to be a `table_content.json` file, per the [Delta Acceptance Testing: Requirements doc](https://docs.google.com/document/d/1ukEHlz7PgH83kRsDJmoBSYCV3yOzJFpl66tfnRUtdAk/edit).  Let me know if this explanation makes more sense.

Reply:
This makes sense. Having a protocol section, or explaining what we are to do with the different generated tables, would help make this clearer however

Reply:
Added a description on what should be done with the different generated tables, let me know if you think that helps.